libs/langchain/langchain/vectorstores/neo4j_vector.py

```diff
@@ -48,14 +48,19 @@ def _get_search_index_query(search_type: SearchType) -> str:
             "CALL { "
             "CALL db.index.vector.queryNodes($index, $k, $embedding) "
             "YIELD node, score "
-            "RETURN node, score UNION "
+            "WITH collect({node:node, score:score}) AS nodes, max(score) AS max "
+            "UNWIND nodes AS n "
+            # We use 0 as min
+            "RETURN n.node AS node, (n.score / max) AS score UNION "
```

Comment:
what happens if max is 0 ? 
e.g. for empty results
let the division by zero error bubble up?

Reply:
Can this even happen? If empty results, then nothing get `UNWIND`-ed?